android/app/build.gradle

```diff
@@ -66,4 +73,8 @@ dependencies {
     implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.2.0'
     implementation 'androidx.lifecycle:lifecycle-viewmodel-ktx:2.2.0'
     implementation 'androidx.activity:activity-ktx:1.1.0'
+    implementation 'junit:junit:4.13.1'
```

Comment:
These could be `androidTestImplementation`, right?

Reply:
Tried that but no. Not sure exactly why but when the test tries to make screenshot, app doesn't come to expected state ("CREATED").